### PR TITLE
Document existing code

### PR DESCRIPTION
**Pull request by Eleonore9**
Before starting adding to the code, document what is already there.

* [x] docstrings
* [x] comments
* [x] unit tests

Solve issues #4 #8

### CODE REVIEW

**Comment by acron0** [comment]
@Eleonore9 Could you add some tests for csv write + read?

**Comment by Eleonore9** [comment]
Oh yes, that's a good point! I can add a test csv.

File: src/schemas.clj [hunk 1]
```diff
@@ -31,3 +38,10 @@
 (defn apply-schema-coercion [data schema]
   {:column-names (apply-col-names-schema schema data)
    :columns (vec (apply-row-schema schema data))})
+
+(def TestData1
```
Comment (acron0) [inline src/schemas.clj]: Can these be removed or moved into a test file?

File: src/schemas.clj [hunk 2]
```diff
@@ -1,6 +1,13 @@
 (ns schemas
```
Comment (acron0) [inline src/schemas.clj]: How come this file isn't in the lapita namespace?

Comment (Eleonore9) [reply src/schemas.clj]: Not sure. I think I just messed up :confused:

File: src/schemas.clj (hunk 1 above)
Comment (Eleonore9) [reply src/schemas.clj]: Sure

**Review by acron0 — APPROVED** [review]

**Pull request merged**